app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import (NotVaccinatedError,
```

Comment:
This is a code style issue. According to the checklist, you should use parentheses `()` while importing multiple classes/modules. Here is how you can do it:

python
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```